src/bindings/python/tests/mock/mock_py_frontend/CMakeLists.txt

```diff
@@ -34,6 +34,13 @@ target_link_libraries(${TARGET_NAME} PUBLIC
 ov_add_library_version(${TARGET_NAME})
 add_clang_format_target(${TARGET_NAME}_clang FOR_TARGETS ${TARGET_NAME})
 
-install(TARGETS ${TARGET_NAME}
-    RUNTIME DESTINATION ${OV_CPACK_RUNTIMEDIR} COMPONENT tests EXCLUDE_FROM_ALL
-    LIBRARY DESTINATION ${OV_CPACK_LIBRARYDIR} COMPONENT tests EXCLUDE_FROM_ALL)
+if(BUILD_SHARED_LIBS)
+    install(TARGETS ${TARGET_NAME}
+        RUNTIME DESTINATION ${OV_CPACK_RUNTIMEDIR} COMPONENT tests EXCLUDE_FROM_ALL
+        LIBRARY DESTINATION ${OV_CPACK_LIBRARYDIR} COMPONENT tests EXCLUDE_FROM_ALL)
+else()
+    set(pyversion python${PYTHON_VERSION_MAJOR}.${PYTHON_VERSION_MINOR})
+    install(TARGETS ${TARGET_NAME}
+        RUNTIME DESTINATION ${OV_CPACK_PYTHONDIR}/${pyversion}/openvino COMPONENT tests EXCLUDE_FROM_ALL
+        LIBRARY DESTINATION ${OV_CPACK_PYTHONDIR}/${pyversion}/openvino COMPONENT tests EXCLUDE_FROM_ALL)
```

Comment:
what's about situations when we run tests against openvino wheel package? I suppose installation to the same place as OV library does not work here. Maybe to install to `tests` folder and use `OV_FRONTEND_PATH`?

Reply:
I'm still checking it. It looks like for static build LD_LIBRARY_PATH or OV_FRONTEND_PATH probably needs to be set

Reply:
`LD_LIBRARY_PATH` has not effect for frontends, FEM searches FEs in the folder with the `openvino` library or from `OV_FRONTEND_PATH`